transport_nantes/topicblog/views.py

```diff
@@ -1080,9 +1080,44 @@ class TopicBlogPressSend(PermissionRequiredMixin, LoginRequiredMixin,
     success_url = "/"
 
 
+class TopicBlogPressIndex(ListView):
+    model = TopicBlogPress
+    template_name = 'topicblog/topicblogpress_index.html'
+    context_object_name = 'press_releases_list'
+    paginate_by = 10
+
+    def get_queryset(self) -> list:
+
+        # Get the list of slugs that have been published at least once
+        all_published_slugs = (
+            TopicBlogPress.objects.filter(publication_date__isnull=False)
```

Comment:
What guarantees that you select the slug with the maximum `publication_date`?

Reply:
And, I see no tests.

Reply:
Did you, by any chance, miss the changes in `test_topic_blog_part_2.py` ? 

Also, the point of the all_published_slugs isn't to get the currently published items, only their slugs. It's a first step.